app/build.gradle

```diff
@@ -293,6 +299,11 @@ android {
             // use the settings from defaultConfig
         }
 
+        managed {
```

Comment:
I needed to release to the Play Store because sending down the configuration `yaml` file only can happen through Play Store. Of course I couldn't do this on our production build so I added this flavor. Could remove now, but I'm sure I'll need to test new features in the future so I opted to leave in